### ISSUE TITLE
Misleading heading in homepage

### ISSUE THREAD

**Issue by bravegnu** [pythonindia/junction#593]
The heading "Conferences" in the homepage is misleading: https://in.pycon.org/cfp/ Every there are multiple links, for submitting proposals. We should come up with something better, or remove it. Or may be be categorize the links by conference, like so:

   * Conferences
      * PyCon India 2019
         * Talks
         * Worksops
         * Posters
         * Devsprints
      * PyCon India 2018
        * Talks
        * Devsprints

**Comment by de-sh** [comment]
True, the app seems to be denoting each cfp as a separate conference
